RiotSwiftUI/Modules/Room/PollHistory/View/SegmentedPicker.swift

```diff
@@ -0,0 +1,61 @@
+// 
+// Copyright 2023 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+struct SegmentedPicker<Tag: Hashable>: View {
```

Comment:
Ah its a shame we can't use a custom `PickerStyle` here to get voiceover support for free. Looks to be a private API for now 😕

Should probably add the necessary accessibility modifiers to the segments to communicate what is/isn't selected for now I guess.

Reply:
> Ah its a shame we can't use a custom PickerStyle here to get voiceover support for free. Looks to be a private API for now 

Yeah, I tried that before I invented a new component.
Apple you need to do the right thing here! 😄

Added the accessibility labels btw: https://github.com/vector-im/element-ios/pull/7267/commits/5735b5d2ce865bd63177304994e60a7bc06d1b72